tests/wpt/metadata-layout-2020/webdriver/tests/execute_script/json_serialize_windowproxy.py.ini

```diff
@@ -0,0 +1,7 @@
+[json_serialize_windowproxy.py]
+  [test_window_open]
+    expected: ERROR
+
+  [test_frame]
+    expected: ERROR
```

Comment:
These were intermittent ERROR 👀 

```
  â–¶ Unexpected subtest result in /webdriver/tests/execute_script/json_serialize_windowproxy.py:
  â”‚ FAIL [expected ERROR] b'test_window_open'
  â”‚   â†’ TypeError: the JSON object must be str, bytes or bytearray, not dict
  â”‚ 
  â”‚ session = <Session 5b120076-fd0d-42e8-af3d-29738ea5c9aa>
  â”‚ 
  â”‚     def test_window_open(session):
  â”‚         # auxiliary browsing context
  â”‚         session.execute_script("window.foo = window.open()")
  â”‚     
  â”‚         response = execute_script(session, "return window.foo;")
  â”‚         raw_json = assert_success(response)
  â”‚     
  â”‚ >       obj = json.loads(raw_json)
  â”‚ 
  â”‚ raw_json   = {'window-fcc6-11e5-b4f8-330a88ab9d7f': '8fb95fadfe85442082bec41c55e17224'}
  â”‚ response   = <Responsetatus=200 body={"value": {"window-fcc6-11e5-b4f8-330a88ab9d7f": "8fb95fadfe85442082bec41c55e17224"}}>
  â”‚ session    = <Session 5b120076-fd0d-42e8-af3d-29738ea5c9aa>
  â”‚ 
  â”‚ tests/wpt/web-platform-tests/webdriver/tests/execute_script/json_serialize_windowproxy.py:39: 
  â”‚ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  â”‚ 
  â”‚ s = {'window-fcc6-11e5-b4f8-330a88ab9d7f': '8fb95fadfe85442082bec41c55e17224'}
  â”‚ cls = None, object_hook = None, parse_float = None, parse_int = None
  â”‚ parse_constant = None, object_pairs_hook = None, kw = {}
  â”‚ 
  â”‚     def loads(s, *, cls=None, object_hook=None, parse_float=None,
  â”‚             parse_int=None, parse_constant=None, object_pairs_hook=None, **kw):
  â”‚         """Deserialize ``s`` (a ``str``, ``bytes`` or ``bytearray`` instance
  â”‚         containing a JSON document) to a Python object.
  â”‚     
  â”‚         ``object_hook`` is an optional function that will be called with the
  â”‚         result of any object literal decode (a ``dict``). The return value of
  â”‚         ``object_hook`` will be used instead of the ``dict``. This feature
  â”‚         can be used to implement custom decoders (e.g. JSON-RPC class hinting).
  â”‚     
  â”‚         ``object_pairs_hook`` is an optional function that will be called with the
  â”‚         result of any object literal decoded with an ordered list of pairs.  The
  â”‚         return value of ``object_pairs_hook`` will be used instead of the ``dict``.
  â”‚         This feature can be used to implement custom decoders.  If ``object_hook``
  â”‚         is also defined, the ``object_pairs_hook`` takes priority.
  â”‚     
  â”‚         ``parse_float``, if specified, will be called with the string
  â”‚         of every JSON float to be decoded. By default this is equivalent to
  â”‚         float(num_str). This can be used to use another datatype or parser
  â”‚         for JSON floats (e.g. decimal.Decimal).
  â”‚     
  â”‚         ``parse_int``, if specified, will be called with the string
  â”‚         of every JSON int to be decoded. By default this is equivalent to
  â”‚         int(num_str). This can be used to use another datatype or parser
  â”‚         for JSON integers (e.g. float).
  â”‚     
  â”‚         ``parse_constant``, if specified, will be called with one of the
  â”‚         following strings: -Infinity, Infinity, NaN.
  â”‚         This can be used to raise an exception if invalid JSON numbers
  â”‚         are encountered.
  â”‚     
  â”‚         To use a custom ``JSONDecoder`` subclass, specify it with the ``cls``
  â”‚         kwarg; otherwise ``JSONDecoder`` is used.
  â”‚         """
  â”‚         if isinstance(s, str):
  â”‚             if s.startswith('\ufeff'):
  â”‚                 raise JSONDecodeError("Unexpected UTF-8 BOM (decode using utf-8-sig)",
  â”‚                                       s, 0)
  â”‚         else:
  â”‚             if not isinstance(s, (bytes, bytearray)):
  â”‚ >               raise TypeError(f'the JSON object must be str, bytes or bytearray, '
  â”‚                                 f'not {s.__class__.__name__}')
  â”‚ E               TypeError: the JSON object must be str, bytes or bytearray, not dict
  â”‚ 
  â”‚ cls        = None
  â”‚ kw         = {}
  â”‚ object_hook = None
  â”‚ object_pairs_hook = None
  â”‚ parse_constant = None
  â”‚ parse_float = None
  â”‚ parse_int  = None
  â”‚ s          = {'window-fcc6-11e5-b4f8-330a88ab9d7f': '8fb95fadfe85442082bec41c55e17224'}
  â”‚ 
  â”” /usr/local/Cellar/python@3.9/3.9.0_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/__init__.py:339: TypeError

  â–¶ Unexpected subtest result in /webdriver/tests/execute_script/json_serialize_windowproxy.py:
  â”‚ FAIL [expected ERROR] b'test_frame'
  â”‚   â†’ TypeError: the JSON object must be str, bytes or bytearray, not dict
  â”‚ 
  â”‚ session = <Session 5b120076-fd0d-42e8-af3d-29738ea5c9aa>
  â”‚ 
  â”‚     def test_frame(session):
  â”‚         # nested browsing context
  â”‚         append = """
  â”‚             window.frame = document.createElement('iframe');
  â”‚             document.body.appendChild(frame);
  â”‚         """
  â”‚         session.execute_script(append)
  â”‚     
  â”‚         response = execute_script(session, "return frame.contentWindow;")
  â”‚         raw_json = assert_success(response)
  â”‚     
  â”‚ >       obj = json.loads(raw_json)
  â”‚ 
  â”‚ append     = ('\n'
  â”‚  "        window.frame = document.createElement('iframe');\n"
  â”‚  '        document.body.appendChild(frame);\n'
  â”‚  '    ')
  â”‚ raw_json   = {'frame-075b-4da1-b6ba-e579c2d3230a': '473ba970ad0f4247a2d6a2dc005347a1'}
  â”‚ response   = <Responsetatus=200 body={"value": {"frame-075b-4da1-b6ba-e579c2d3230a": "473ba970ad0f4247a2d6a2dc005347a1"}}>
  â”‚ session    = <Session 5b120076-fd0d-42e8-af3d-29738ea5c9aa>
  â”‚ 
  â”‚ tests/wpt/web-platform-tests/webdriver/tests/execute_script/json_serialize_windowproxy.py:57: 
  â”‚ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
  â”‚ 
  â”‚ s = {'frame-075b-4da1-b6ba-e579c2d3230a': '473ba970ad0f4247a2d6a2dc005347a1'}
  â”‚ cls = None, object_hook = None, parse_float = None, parse_int = None
  â”‚ parse_constant = None, object_pairs_hook = None, kw = {}
  â”‚ 
  â”‚     def loads(s, *, cls=None, object_hook=None, parse_float=None,
  â”‚             parse_int=None, parse_constant=None, object_pairs_hook=None, **kw):
  â”‚         """Deserialize ``s`` (a ``str``, ``bytes`` or ``bytearray`` instance
  â”‚         containing a JSON document) to a Python object.
  â”‚     
  â”‚         ``object_hook`` is an optional function that will be called with the
  â”‚         result of any object literal decode (a ``dict``). The return value of
  â”‚         ``object_hook`` will be used instead of the ``dict``. This feature
  â”‚         can be used to implement custom decoders (e.g. JSON-RPC class hinting).
  â”‚     
  â”‚         ``object_pairs_hook`` is an optional function that will be called with the
  â”‚         result of any object literal decoded with an ordered list of pairs.  The
  â”‚         return value of ``object_pairs_hook`` will be used instead of the ``dict``.
  â”‚         This feature can be used to implement custom decoders.  If ``object_hook``
  â”‚         is also defined, the ``object_pairs_hook`` takes priority.
  â”‚     
  â”‚         ``parse_float``, if specified, will be called with the string
  â”‚         of every JSON float to be decoded. By default this is equivalent to
  â”‚         float(num_str). This can be used to use another datatype or parser
  â”‚         for JSON floats (e.g. decimal.Decimal).
  â”‚     
  â”‚         ``parse_int``, if specified, will be called with the string
  â”‚         of every JSON int to be decoded. By default this is equivalent to
  â”‚         int(num_str). This can be used to use another datatype or parser
  â”‚         for JSON integers (e.g. float).
  â”‚     
  â”‚         ``parse_constant``, if specified, will be called with one of the
  â”‚         following strings: -Infinity, Infinity, NaN.
  â”‚         This can be used to raise an exception if invalid JSON numbers
  â”‚         are encountered.
  â”‚     
  â”‚         To use a custom ``JSONDecoder`` subclass, specify it with the ``cls``
  â”‚         kwarg; otherwise ``JSONDecoder`` is used.
  â”‚         """
  â”‚         if isinstance(s, str):
  â”‚             if s.startswith('\ufeff'):
  â”‚                 raise JSONDecodeError("Unexpected UTF-8 BOM (decode using utf-8-sig)",
  â”‚                                       s, 0)
  â”‚         else:
  â”‚             if not isinstance(s, (bytes, bytearray)):
  â”‚ >               raise TypeError(f'the JSON object must be str, bytes or bytearray, '
  â”‚                                 f'not {s.__class__.__name__}')
  â”‚ E               TypeError: the JSON object must be str, bytes or bytearray, not dict
  â”‚ 
  â”‚ cls        = None
  â”‚ kw         = {}
  â”‚ object_hook = None
  â”‚ object_pairs_hook = None
  â”‚ parse_constant = None
  â”‚ parse_float = None
  â”‚ parse_int  = None
  â”‚ s          = {'frame-075b-4da1-b6ba-e579c2d3230a': '473ba970ad0f4247a2d6a2dc005347a1'}
  â”‚ 
  â”” /usr/local/Cellar/python@3.9/3.9.0_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/json/__init__.py:339: TypeError
```